kubernetes/cmsweb/services/wmarchive.yaml

```diff
@@ -60,6 +60,12 @@ spec:
         name: wmarchive
         args:
           - /data/run.sh
+        livenessProbe:
```

Comment:
You duplicated livenessProbe in wmarchive which I added before, see line 76. Please check and fix.